src/basic/os-util.c

```diff
@@ -225,6 +285,135 @@ int open_extension_release(const char *root, const char *extension, bool relax_e
         return 0;
 }
 
+int open_configuration_release(const char *root, const char *configuration, bool relax_configuration_release_check, char **ret_path, int *ret_fd) {
+        _cleanup_free_ char *q = NULL;
+        int r, fd;
+
+        if (configuration) {
+                const char *configuration_full_path;
+
+                if (!image_name_is_valid(configuration))
+                        return log_debug_errno(SYNTHETIC_ERRNO(EINVAL),
+                                               "The extension name %s is invalid.", configuration);
+
+                configuration_full_path = strjoina("/usr/lib/configuration-release.d/configuration-release.", configuration);
```

Comment:
syscfg images are only supposed to contain /etc/, hence using a path in /usr/ib/ makes no sense here.

i guess the release info for sysext images really must be /etc/syscfg-release.d/syscfg-release.<suffix>`